pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -91,6 +91,8 @@
 
 def stopping_condition(op: Operator) -> bool:
     """A function that determines whether or not an operation is supported by ``lightning.gpu``."""
+    if op.name == "C(SProd)":
+        return True
```

Comment:
Hmm... it was a quick fix for Isaac discussed in https://github.com/PennyLaneAI/pennylane-lightning/issues/900, now I remember. I wonder if there is a more generic solution for Lightning?

Reply:
Alright! I think that's the most straightforward approach to fix this issue. Happy to approve this!